oop_Session1/TestDOM.java

```diff
@@ -0,0 +1,121 @@
+package oop_Session1;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class TestDOM {
+
+	    static DOM dom;
+		static CompositeElement div1;

```

Comment:
Only the `DOM` element needs to be declared globally; other elements can be declared within the `@Before` method as they are not referred to explicitly in the test methods.